documentation/modules/ref-brokered-infra-config-fields.adoc

```diff
@@ -0,0 +1,65 @@
+// Module included in the following assemblies:
+//
+// master-openshift.adoc
+// master-kubernetes.adoc
+// master.adoc
+
+[id='ref-brokered-infra-config-fields-{context}']
+= Brokered infrastructure configuration fields
+
+This table shows the fields available for the brokered infrastructure configuration and a brief description.
+
+.Brokered infrastructure configuration fields table
+[cols="40%a,60%a,options="header"]
+|===
+|Field |Description
+|version |Specifies the {ProductName} version used. When upgrading, {ProductName} uses this field to determine whether to upgrade the infrastructure to the requested version.
+|admin.resources.memory |Specifies the amount of memory allocated to the admin pod.
+|admin.podTemplate.metadata.labels |Specifies the labels added to the admin pod.
+
+ifeval::["{cmdcli}" == "kubectl"]
```

Comment:
@lulf Is there a more elegant way to handle this? If not, I'll fix the other table the same way.

Reply:
Yeah, replace Kubernetes with the {KubePlatform} attribute

Reply:
@lulf Right! Thank you :)